src/api/integrations/ordnance-survey/index.ts

```diff
@@ -0,0 +1,48 @@
+import axios from 'axios';
+import dotenv from 'dotenv';
+import { OrdnanceSurveyAPIResponse } from '../../types';
+import { ORDNANCE_SURVEY_QUERY_URL } from '../../constants';
+
+dotenv.config();
+
+const ordnanceSurveyBaseUrl = process.env.ORDNANCE_SURVEY_API_URL;
+const ordnanceSurveyApiKey = process.env.ORDNANCE_SURVEY_API_KEY;
```

Comment:
Shall we destructure the env vars instead of creating a new variable?

```js
const { ORDNANCE_SURVEY_API_URL, ORDNANCE_SURVEY_API_KEY} = process.env;
```